.github/workflows/copilot-setup-steps.yml

```diff
@@ -0,0 +1,42 @@
+name: "Copilot Setup Steps"
+
```

Comment:
I'd like some general comment here about what this is for, ideally including a link to the corresponding github docs